src/Microsoft.Framework.Runtime.Interfaces/Microsoft.Framework.Runtime.Interfaces.xproj

```diff
@@ -10,8 +10,11 @@
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
     <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
   </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <RootNamespace>Microsoft.Framework.Runtime</RootNamespace>
```

Comment:
Do we or do we not want VS to use `Microsoft.Framework.Runtime` as the default base namespace for types in `Microsoft.Framework.Runtime.Interfaces`? As I said above, without this change it uses the folder name as the base, which means devs have to remember to change it.